server/src/storage/mod.rs

```diff
@@ -74,11 +75,12 @@ pub struct StorageEngine {
     /// Manager that contains and controls all uncompressed data.
     uncompressed_data_manager: UncompressedDataManager,
     /// Manager that contains and controls all compressed data.
-    compressed_data_manager: CompressedDataManager,
+    pub compressed_data_manager: CompressedDataManager,
```

Comment:
Could this be private if a method is added to `StorageEngine`?

Reply:
Fixed.